ios/PurchasesHybridCommon/PurchasesHybridCommon/RCTransaction+HybridAdditions.m

```diff
@@ -9,13 +9,15 @@
 #import "RCTransaction+HybridAdditions.h"
 #import "NSDate+HybridAdditions.h"
 
-@implementation RCTransaction (RCPurchases)
+@implementation RCStoreTransaction (RCPurchases)
 
 - (NSDictionary *)dictionary
 {
     NSMutableDictionary *d = [NSMutableDictionary dictionaryWithDictionary:@{
-        @"revenueCatId": self.revenueCatId,
-        @"productId": self.productId,
+        // JOSH: is transactionIdentifier acutaly revenueCatId?????
+        // JOSH: do we need to change these keys?
+        @"revenueCatId": self.transactionIdentifier,
+        @"productId": self.productIdentifier,
```

Comment:
Can the keys of this dictionary change? I'm not super certain what this dictionary is used for and what relies on these keys 🤔

Reply:
These are sent down to the hybrids. Changing them has some implications:
1. In JS based hybrid SDKs, the keys are directly transformed into a Javascript objects, so changing this will break implementations.
1. In Flutter and Unity we convert manually to Dart and C# objects. For example: https://github.com/RevenueCat/purchases-flutter/blob/amazon-latest/lib/transaction.dart#L15 https://github.com/RevenueCat/purchases-unity/blob/main/RevenueCat/Scripts/Transaction.cs#L32

Let me know if you have more questions